front_materials/dedication.tex

```diff
@@ -1 +1,5 @@
-Put your dedication text here.
\ No newline at end of file
+\begin{center}
+
+Put your dedication text here.
+
+\end{center}
```

Comment:
Can you place the centering in the .cls file rather than in the dedication, please?